### PR TITLE
Fix presto initialization error due to alluxio connector

### PR DESCRIPTION
**Pull request by yuzhu**
Before this fix, presto would fail to start on emr 5.30 or above due to class conflicts, because we are using alluxio bundled connector. 
Switch to using presto's  built-in connector

### CODE REVIEW

File: integration/emr/alluxio-emr.sh [hunk 1]
```diff
@@ -668,9 +668,10 @@ IN
   local -r emr_version=$(jq ".releaseLabel" /mnt/var/lib/info/extraInstanceData.json  | sed -e 's/^"emr-//' -e 's/"$//')
```
Comment (Xenorith) [inline integration/emr/alluxio-emr.sh:668]: update the comments before this line? and explain why the behavior for 5.30+ is different

File: integration/emr/alluxio-emr.sh [hunk 2]
```diff
@@ -668,9 +668,10 @@ IN
...
   local -r emr_major=$(echo "${emr_version}" | sed -s 's/\([[:digit:]]\+\)\.\([[:digit:]]\+\)\.[[:digit:]]\+/\1/')
   local -r emr_minor=$(echo "${emr_version}" | sed -s 's/\([[:digit:]]\+\)\.\([[:digit:]]\+\)\.[[:digit:]]\+/\2/')
-  if [ "${emr_major}" -gt 5 ] || [[ "${emr_major}" -eq 5 && "${emr_minor}" -ge 28 ]]; then
+  if [[ "${emr_major}" -eq 5 && "${emr_minor}" -ge 28 && "${emr_minor}" -lt 30  ]]; then
```
Comment (Xenorith) [inline integration/emr/alluxio-emr.sh:671]: wow so we are only doing this for emr 5.28 and 5.29, such a narrow scope

**Review by Xenorith — APPROVED** [review]
nit to update comment otherwise good

**Review by Xenorith — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**